package.json

```diff
@@ -23,7 +23,8 @@
   ],
   "scripts": {},
   "peerDependencies": {
-    "eslint": ">=8"
+    "eslint": ">=8",
+    "eslint-plugin-distributive": "@distributive/eslint-plugin"
```

Comment:
**issue:** The issue highlighted in https://github.com/Distributive-Network/eslint-config/pull/45#discussion_r1267134139 still occurs. At the very least, this pattern (renaming?) isn't common when specifying peer dependencies.

```suggestion
    "@distributive/eslint-plugin": "^1.0.0",
    "eslint": ">=8"
```